feature-libs/cart/main/core/services/product-import-info.service.ts

```diff
@@ -1,43 +1,27 @@
-import { isDevMode } from '@angular/core';
+import { Injectable, isDevMode } from '@angular/core';
 import { ofType } from '@ngrx/effects';
 import { ActionsSubject } from '@ngrx/store';
-import { CartActions } from '@spartacus/cart/main/core';
-import { Observable } from 'rxjs';
-import { filter, map, switchMap, take } from 'rxjs/operators';
 import {
-  ProductData,
   ProductImportInfo,
   ProductImportStatus,
 } from '@spartacus/cart/main/root';
+import { Observable } from 'rxjs';
+import { filter, map } from 'rxjs/operators';
+import { CartActions } from '../store/actions';
 
-/**
- * An abstract class for contexts based on cart (active cart, saved cart) which share part of logic - mainly handle cart actions and map results.
- */
-export abstract class CartOrderEntriesContext {
+@Injectable({
+  providedIn: 'root',
+})
+export class ProductImportInfoService {
```

Comment:
Does't removing CartOrderEntriesContext prevent the `AddEntries` method from beiing reused?